roles/ovirt-datacenter-cleanup/tasks/main.yml

```diff
@@ -0,0 +1,128 @@
+# Cleanup VMPools
+- name: Find existing VMPools
+  ovirt_vmpools_facts:
+    auth: "{{ ovirt_auth }}"
+    pattern: "datacenter={{ data_center_name }}"
+    fetch_nested: true
+
+- name: Show existing VMPools
+  debug:
+    var: ovirt_vm_pools
```

Comment:
Maybe show just names? or only relevant information not all?

Reply:
I think I can skip completely these debug messages.